docs/dictionary/command/revBrowserAddJavaScriptHandler.lcdoc

```diff
@@ -17,39 +17,32 @@ Security: network
 Example:
 local tBrowserID
 put revOpenBrowserCef(the windowId of this stack, "http://www.livecode.com") into tBrowserID
-revBrowserAddJavaScriptHander, tBrowserID, "myLiveCodeHandler"
+revBrowserAddJavaScriptHandler, tBrowserID, "myLiveCodeHandler"
```

Comment:
I guess it's not actually a syntax error though is it? Just an empty first argument.